.buildkite/pipeline.yml

```diff
@@ -138,21 +138,20 @@ steps:
   - label: ":jenkins: Release - Package Registry Distribution"
     key: "release-package-registry"
     trigger: "package-registry-release-package-registry-distribution"
+    async: true
     build:
       branch: "main"
       meta_data:
         DOCKER_TAG: "${BUILDKITE_TAG}"
     if: "build.env('BUILDKITE_TAG') != ''"
-    depends_on:
-      - step: "release-test"
-        allow_failure: false
 
   - trigger: "fleet-server-package-mbp"
     label: ":esbuild: Downstream - Package"
     key: "downstream-package"
+    async: true
     if: "build.env('BUILDKITE_PULL_REQUEST') == 'false' && build.env('BUILDKITE_TAG') == '' && build.env('BUILDKITE_BRANCH') != ''"
     build:
       branch: "${BUILDKITE_BRANCH}"
     depends_on:
-      - step: "release-package-registry"
+      - step: "release-test"
```

Comment:
Change this dependency, to be the one set in `release-package-registry` to continue with the same chain/flow of steps.